content/applications/hr/appraisals/new_appraisals.rst

```diff
@@ -0,0 +1,283 @@
+==============
+New appraisals
+==============
+
+To create a new appraisal for an employee, first navigate to the main *Appraisals* dashboard by
+opening the :menuselection:`Appraisals` app. The :guilabel:`Appraisals` dashboard is the default
+view.
+
+Appraisals dashboard
+====================
+
+All appraisals are displayed on the dashboard in a default Kanban view, with a list of groupings on
+the left side of the dashboard, including :guilabel:`Company`, :guilabel:`Department`, and
+:guilabel:`Status`.
+
+Click any grouping option to view appraisals for **only** the chosen selection.
+
+.. note::
+   Only groupings with multiple selections appear in the list. For example, if a database only has
+   one company, the :guilabel:`Company` grouping does **not** appear, since there is no other
+   company to select.
+
+Each appraisal card displays the following information:
+
+- :guilabel:`Name`: the employee's name.
+- :guilabel:`Department`: the department the employee is associated with.
+- :guilabel:`Company`: the company the employee works for. This only appears in a multi-company
+  database.
+- :guilabel:`Date`: the date the appraisal was requested, or is scheduled for in the future.
+- :guilabel:`Activities`: any activities that are scheduled for the appraisal, such as
+  :guilabel:`Meetings` or :guilabel:`Phone Calls.` If no activities are scheduled, an activity can
+  be scheduled from the :icon:`fa-clock` :guilabel:`(clock)` icon, located in the bottom-left corner
+  of an appraisal card.
+- :guilabel:`Manager`: the employee's manager, indicated by the profile icon in the bottom-right
+  corner of an appraisal card.
+- :guilabel:`Status banner`: the status of the appraisal. A banner appears if an appraisal is
+  marked as either :guilabel:`Canceled` or :guilabel:`Done`. If no banner is present, that means the
+  appraisal has not happened, or has not been scheduled yet.
```

Comment:
I agree with KSC here, and suggest the following since there are no field names visible for a guilabel.

Also linking to the activities doc instead of explaining how to schedule an activity from this view, which breaks the flow IMO
```suggestion
- **Name**: the employee's name.
- **Department**: the department the employee is associated with.
- **Company**: the company the employee works for. This only appears in a multi-company database.
- **Date**: the date the appraisal was requested, or is scheduled for in the future.
- **Activities**: any :doc:`activities <../../essentials/activities>` that are scheduled for the
  appraisal, such as *Meetings* or *Phone Calls*.
- **Manager**: the employee's manager, indicated by the profile icon in the bottom-right corner of
  an appraisal card.
- **Status banner**: the status of the appraisal. A banner appears if an appraisal is marked as
  either *Canceled* or *Done*. If no banner is present, that means the appraisal has not happened,
  or has not been scheduled yet.
```